Renderer.js

```diff
@@ -0,0 +1,52 @@
+
+class Renderer {
+constructor(apiManager){
+    this.apiManager=apiManager
+}
```

Comment:
The render class (aka the View) should not have direct access to API manager (aka the Model)